### PR TITLE
Bookbuilding layout fixes

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/shared/forms/formField/FormField.tsx [hunk 1]
```diff
@@ -1,129 +1,18 @@
...
-                  )}
-              </>
-            );
-          }}
-        />
-      </FormGroup>
-    );
-  }
-}
+export const FormField: React.SFC<FieldGroupProps> = ({ label, ...inputProps }) => (
```
Comment (desfero) [inline app/components/shared/forms/formField/FormField.tsx]: @Moejoe90 I have simplified `FormField` by moving input related staff to separate component. But still, we need to do refactoring as `FormInput` and `FormFieldRaw` contain mainly the same logic for input layout.

Comment (Moejoe90) [reply app/components/shared/forms/formField/FormField.tsx]: Sounds good to me

Comment (desfero) [reply app/components/shared/forms/formField/FormField.tsx]: Ok, i will do some story to refactor it and probably event extract all layout simple forms (not formik one) to separate folder in near future

File: app/components/eto/overview/EtoOverviewStatus/EtoOverviewStatus.tsx [hunk 2]
```diff
@@ -290,17 +290,16 @@ const EtoOverviewStatus: React.SFC<
...
-                minPledge={props.campaigningWidget.minPledge}
-                maxPledge={props.campaigningWidget.maxPledge}
-                isActivated={props.campaigningWidget.isActivated}
-                quote={props.campaigningWidget.quote}
-                investorsLimit={props.campaigningWidget.investorsLimit}
-              />
-            )}
+          {props.status === "campaigning" && (
+            <CampaigningWidget
+              etoId={props.etoId}
```
[the remark this answers is not in the document]
Comment (desfero) [reply app/components/eto/overview/EtoOverviewStatus/EtoOverviewStatus.tsx]: Sure, make sense. Let's, for now, merge it as it is and I will do that when will touch EtoOverviewStatus next time to not block merging

**Pull request merged**